source/includes/documentation/_vue_js_elements.md

```diff
@@ -2964,3 +2964,98 @@ Available methods in Java JDI Light:
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
 
+### 5.41 Checkbox
+
+[Checkbox Vuetify documentation page](https://vuetifyjs.com/en/components/checkboxes/)
+
+Checkbox is located in the following class:
+- __Java__: _com.epam.jdi.light.vuetify.elements.common.Checkbox.java_
+
+```java
+    @Test(description = "Test checks checkbox labels, and array texts")
+    public void modelAsArrayCheckboxesTest() {
+        modelAsArrayCheckboxes.get(1).is().checked();
+        modelAsArrayCheckboxes.get(2).is().unchecked();
+        modelArray.has().text("[ \"" + modelAsArrayCheckboxes.get(1).labelText() + "\" ]");
+        modelAsArrayCheckboxes.get(2).check();
+        modelArray.has().text("[ \"John\", \"Jacob\" ]");
+        modelAsArrayCheckboxes.get(1).uncheck();
+        modelArray.has().text("[ \"Jacob\" ]");
+        modelAsArrayCheckboxes.get(2).uncheck();
+        modelArray.has().text("[]");
+    }
+```
+__Checkboxes__ - The `v-checkbox` component provides users the ability to choose between two distinct values.
+These are very similar to a switch and can be used in complex forms and checklists.
+A simpler version, `v-simple-checkbox` is used primarily as a lightweight alternative in data-table components to select rows or display inline boolean data.
+
+![Checkbox example](../../images/vuetify/checkbox.png)
+
+__Vuetify v2.6.14__ code example:
+```html
+<template>
+  <v-card flat>
+    <v-card-text>
+      <v-container fluid>
+        <v-row>
+          <v-col
+            cols="12"
+            sm="4"
+            md="4"
+          >
+            <v-checkbox
+              v-model="ex4"
+              label="red"
+              color="red"
+              value="red"
+              hide-details
+            ></v-checkbox>
+            <v-checkbox
+              v-model="ex4"
+              label="red darken-3"
+              color="red darken-3"
+              value="red darken-3"
+              hide-details
+            ></v-checkbox>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-card-text>
+  </v-card>
+</template>
+```
+
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 
![image](https://user-images.githubusercontent.com/55189259/212974090-92f5a613-5639-45a9-a34d-d26a5cf022eb.png)